infrastructure/variables.tf

```diff
@@ -24,40 +24,40 @@ variable "subscription" {
   type = string
 }
 
-variable "ilbIp" {
-  type = string
-}
+# variable "ilbIp" {
+#   type = string
+# }
 
 variable "common_tags" {
   type = map(string)
 }
 
-variable "appinsights_instrumentation_key" {
-  type    = string
-  default = ""
-}
+# variable "appinsights_instrumentation_key" {
+#   type    = string
+#   default = ""
+# }
 
-variable "root_logging_level" {
-  type    = string
-  default = "INFO"
-}
+# variable "root_logging_level" {
+#   type    = string
+#   default = "INFO"
+# }
 
-variable "log_level_spring_web" {
-  type    = string
-  default = "INFO"
-}
+# variable "log_level_spring_web" {
+#   type    = string
+#   default = "INFO"
+# }
 
-variable "log_level_rd" {
-  type    = string
-  default = "INFO"
-}
+# variable "log_level_rd" {
+#   type    = string
+#   default = "INFO"
+# }
 
 variable "team_name" {
   type    = string
   default = "RD"
 }
 
-variable "managed_identity_object_id" {
-  type    = string
-  default = ""
-}
\ No newline at end of file
+# variable "managed_identity_object_id" {
+#   type    = string
+#   default = ""
+# }
```

Comment:
can we remove the variables if they are not used?

Reply:
Vars removed now